scripts/permissions_cleanup.py

```diff
@@ -39,7 +39,7 @@ def cleanup_permissions():
     sm.get_session.commit()
 
     pvms = sm.get_session.query(sm.permissionview_model).all()
-    print('STage 1: # of permission view menues is: {}'.format(len(pvms)))
+    print("STage 1: # of permission view menues is: {}".format(len(pvms)))
```

Comment:
Nit. Could you also rename `STage` to `Stage`.

Reply:
Sure. Will do!